packages/rancher-logging/generated-changes/patch/Chart.yaml.patch

```diff
@@ -10,21 +10,26 @@
 +  catalog.cattle.io/namespace: cattle-logging-system
 +  catalog.cattle.io/permits-os: linux,windows
 +  catalog.cattle.io/provides-gvr: logging.banzaicloud.io.clusterflow/v1beta1
-+  catalog.cattle.io/rancher-version: '>= 2.8.0-0 < 2.9.0-0'
++  catalog.cattle.io/rancher-version: '>= 2.9.0-0 < 2.10.0-0'
 +  catalog.cattle.io/release-name: rancher-logging
 +  catalog.cattle.io/type: cluster-tool
 +  catalog.cattle.io/ui-component: logging
-+  catalog.cattle.io/upstream-version: 3.17.10
- apiVersion: v1
- appVersion: 3.17.10
--description: A Helm chart to install Banzai Cloud logging-operator
++  catalog.cattle.io/upstream-version: 4.4.0
+ apiVersion: v2
+ appVersion: 4.4.0
+ description: Logging operator for Kubernetes based on Fluentd and Fluentbit.
+ home: https://kube-logging.github.io
++icon: file://assets/logos/rancher-logging.svg
+ keywords:
+ - logging
+ - fluentd
+ - fluentbit
+ kubeVersion: '>=1.22.0-0'
 -name: logging-operator
-+description: Collects and filter logs using highly configurable CRDs. Powered by Banzai
-+  Cloud Logging Operator.
-+icon: https://charts.rancher.io/assets/logos/logging.svg
-+keywords:
-+- logging
-+- monitoring
-+- security
 +name: rancher-logging
- version: 3.17.10
+ sources:
+ - https://github.com/kube-logging/logging-operator
+ - https://github.com/kube-logging/helm-charts/tree/main/charts/logging-operator
+ type: application
+-version: 4.4.0
++version: 104.0.0+up4.4.0
```

Comment:
104 is for Rancher v2.9. Please use 103 for Rancher v2.8

Reply:
Teams don't usually patch this. We can see that this value is replaced in the generated chart.yaml file with the package version.